source/includes/_plans.md

```diff
@@ -508,4 +506,65 @@ discount |`number`| Between 0.0 an 1.0, where 1.0 is 100% discount on the `total
 total_amount |`number`| The total cost of the subscription payment. If set, the total_amount is not based on the sum of products.
 recurring_days | `array` | When updating the recurring days you can return both of following format:
  - `{"day": 0, "starting_day": {"$date": 1496410494652}, "next_day": {"$date": 1496410494652}}` - it means this recurring day already exists
- - `{"day": 2, "hour": 10, "minute": 15}`
\ No newline at end of file
+ - `{"day": 2, "hour": 10, "minute": 15}`
+ 
+ 
+ ### Get Subscription used by a Plan
+
+You can know if your plan is used in [`Subscription`](#subscriptions) by using the following request which will return you a list of [`Subscription`](#subscriptions).
+Allowing you to get all [`Subscription`](#subscriptions) or some by using filter like `status` or `method` as a parameters.
```

Comment:
We also allow you to use query parameters. Please take a look below for a list of accepted query parameters.